bench.py

```diff
@@ -100,26 +100,21 @@ def run_bench(verbose):
 
 if __name__ == "__main__":
     if len(sys.argv) != 2:
-        print("Usage: %s (llvmlite|llvmpy)"
+        print("Usage: %s llvmlite"
```

Comment:
This file was for comparing llvmpy (https://github.com/llvmpy/llvmpy) with llvmlite. I think we can remove this file all together now.

Reply:
Done.